types/module/configurator.go

```diff
@@ -0,0 +1,43 @@
+package module
```

Comment:
Maybe this code is old code that can be removed and updated to what's in master?

Reply:
You mean using the new `Configurator` interface from `types/module/server` instead? That would mean temporarily adding some public `NewConfigurator` function so that we can use it in `app.go` `app.mm.RegisterServices(servermodule.NewConfigurator(...))` which does not seem to be the primary purpose of this new `configurator` type but could work for now, are we ok with this?

Reply:
I think that could work... but isn't ideal either. I would defer to whatever you think is the best path to be honest...

Reply:
Fully migrating the group module to ADR 033 will allow us to use the new module/service registration instead of `app.mm.RegisterServices(...)` (ref: https://github.com/regen-network/regen-ledger/pull/154#discussion_r564444402) so I'll just delete this `types/module/configurator.go` which won't be needed anymore.